src/components/Buttons/Buttons.css

```diff
@@ -0,0 +1,22 @@
+.orange-button {
+  width: 13.625em;
+  height: 4em;
+  background: #ffa001;
+  border-radius: 2.438em;
+  font-family: Poppins;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5em;
+  line-height: 2.25 em;
+  color: #ffffff;
+  border: 0px;
+}
```

Comment:
Great job using `em` for the units! I definitely need to be better about that. I'm wondering if we could remove `font-style`, `font-weight`, and `line-height`? I think those might already be the default styling. 

I'm also thinking the button might be a little too big. In the screenshot I did this styling to make it a little smaller: 

```
font-size: 1em; 
width: 9.625em;
height: 3.18em; 
```

![image](https://user-images.githubusercontent.com/46573212/155939933-f67d5bf9-8b9d-4adc-9ccf-9bc60f434d7a.png)

Let me know what you think though!

Reply:
I think that works much better. Thank you, Viviana. I was using the values exactly from Figma, and then translated them to em, but I think I did it based on 16px instead of 24. I updated with those changes and removed the default, already baked in.